### ISSUE TITLE
DCGAN

### ISSUE THREAD

**Issue by iamaaditya** [mevender/drgm#1]
I don't know if the goal is to replicate DCGAN true to its original implementation. In anycase DCGAN uses TanH for the final layer activation. I see that you are using [softmax](https://github.com/mevender/drgm/blob/8280c2894c0b2bfcba1255339170f2984520d774/Models/ProtoArchitectures.py#L26).
[Reference implementation](https://github.com/carpedm20/DCGAN-tensorflow/blob/master/model.py#L396) (I can attest to this reference, as I have contributed and also tested this code).

**Comment by iamaaditya** [comment]
Input space is [0,1].
